readme.md

```diff
@@ -0,0 +1 @@
+#Read me!!!
```

Comment:
Please add a space between "#" and "R" and it should have an empty line at the end